modules/__tests__/describePopState.js

```diff
@@ -14,11 +14,17 @@ const describePopState = (createHistory) => {
     })
 
     it('is called when browser navigation is used', (done) => {
-      unlisten = history.listenBefore(() => {
-        done()
-      })
+      setTimeout(() => {
```

Comment:
That makes sense. Do you have a commit that demonstrates this failure? I'd like to take a closer look when I get a chance.